kmod/igb/igb_ptp.c

```diff
@@ -369,17 +365,14 @@ static int igb_ptp_gettime64_i210(struct ptp_clock_info *ptp,
 
 #ifdef HAVE_PTP_CLOCK_INFO_GETTIME64
 static int igb_ptp_settime64_82576(struct ptp_clock_info *ptp,
-				 const struct timespec *ts64)
+				   const struct timespec *ts64)
```

Comment:
Eventually, yes, but the history of timespec64 makes me think this won’t cause a failure and unless it does it’s not my highest priority. If nothing awful happens in the next week, I’ll have a pull request in a few days.

Todd Fujinaka
Software Application Engineer
Networking Division (ND)
Intel Corporation
todd.fujinaka@intel.com
(503) 712-4565

From: andrew-elder [mailto:notifications@github.com]
Sent: Monday, August 24, 2015 6:07 AM
To: AVnu/Open-AVB
Cc: Fujinaka, Todd
Subject: Re: [Open-AVB] igb: fix build on 32-bit kernels (#267)


In kmod/igb/igb_ptp.c<https://github.com/AVnu/Open-AVB/pull/267#discussion_r37748427>:

> @@ -369,17 +365,14 @@ static int igb_ptp_gettime64_i210(struct ptp_clock_info *ptp,

>

>  #ifdef HAVE_PTP_CLOCK_INFO_GETTIME64

>  static int igb_ptp_settime64_82576(struct ptp_clock_info *ptp,

> -                             const struct timespec *ts64)

> +                               const struct timespec *ts64)

@htfujina<https://github.com/htfujina> - will there be a new pull request to fix this then?

—
Reply to this email directly or view it on GitHub<https://github.com/AVnu/Open-AVB/pull/267/files#r37748427>.